src/components/Modal/Modal.tsx

```diff
@@ -141,6 +141,13 @@ export const ModalForwardRef: React.ForwardRefRenderFunction<
   const ariaLabelledBy = divProps['aria-labelledby']
   const ariaDescribedBy = divProps['aria-describedby']
 
+  if (undefined === ariaLabelledBy) {
+    console.error(`${id} is missing aria-labelledby attribute`)
+  }
+  if (undefined === ariaDescribedBy) {
+    console.error(`${id} is missing aria-describedby attribute`)
+  }
```

Comment:
Any reason not to check for general falsiness so that we also check for null values? That's what USWDS is doing (https://github.com/uswds/uswds/pull/5315/files#diff-c7fd10b2638da39aa2cdf0583b9f14f1b05d5451909df6bbcd5ec528d74102afR223-R227)

```suggestion
  if (!ariaLabelledBy) {
    console.error(`${id} is missing aria-labelledby attribute`)
  }
  if (!ariaDescribedBy) {
    console.error(`${id} is missing aria-describedby attribute`)
  }
```

Reply:
I swear I did that at first and it somehow failed tests, but now they pass. Anyway, fixed 🤷